README.md

```diff
@@ -66,67 +93,27 @@ You are now ready to start writing some code! If you dont already have a favorit
-
-linter-eslint - hooks in with "lint" rules and helps you avoid mistakes and typos as well as adhering to coding best practice
-
-Something for jsdocs (comment blocks that go above methods) I am currently using `atom-easy-jsdoc` which helps a bit but there are many out there
-
-Atom already understands js, jsx and json files so theres not much to do there but there are many many helpful plugins available in atom to explore.
-
-### Create a branch
-
-The first thing to do before we start coding is to checkout a new branch of the code. This way you can work on changes independently and not affect the code in the `master` branch. In your terminal (while in the project folder) type:
+Regardless of what editor you are using you will want it to be ready for coding a react / node javascript application so you will want plugins related to syntax highlighting and inline helpers for those languages.
 
 ```
-git checkout -b <a-reasonable-name>
+git checkout -b winternship2020
 ```
 
-replacing the `<a-reasonable-name>` bit with a branch name of your choosing (since we dont know exactly what well be doing yet anything reasonably unique will do)
-
-### Explore the basic parts of the application
-
-Lets have a quick look at the basic parts of the application and their related dependencies
-
-#### package.json
-
-The package file is something you will see in almost every javascript application you will come across. This file describes the application, its dependencies and defines scripts used for building and running our application. It hooks us up to `npm` and allows us to install and manage dependencies. `npm` is a very widely used network of open source javascript packages. If you can think of it, it is likely that something exists there for you to install, and if not you can create something and make it available for others. We wont be covering that in this exercise but if curious we can look into that process.
-
-https://www.npmjs.com/
-
-#### server.js
-
-This is the server side of our application. It is based on `express` which is a very common framework for creating node js web applications. See the documentation here:
-
-https://expressjs.com/
-
-This file is what is executed when we run `npm start` you can see the reference to that inside the `package.json` under `scripts`. This runs `node` on the server.js file which creates an express application and tells it to start "listening" on port 3000. It is here that we can start creating `routes` for our application to call and also where we will be interacting with the shutterstock api via the shutterstock "sdk" (software development kit) The sdk provides an easy way to interact with the api by setting up our authentication and providing validated methods we can use to call the various endpoints it provides.
-
-https://github.com/shutterstock/public-api-javascript-sdk
+### Let's write some code
```

Comment:
added this section for the first code change. I stuck it prior to the `Understanding` parts because I thought it might be nice for them to feel like they made a change before getting any more from the the firehose of knowledge